client/src/pages/login/index.js

```diff
@@ -7,27 +8,79 @@ import "./login.css";
 class Login extends Component {
   state = {
     isLogin: false,
+    alert: null,
     loginInfo: {}
   };
 
+  errorPassOrUsername() {
+    const getAlert = () => (
+      <SweetAlert
+        warning
+        title="Wrong!"
+        confirmBtnBsStyle="danger"
+        cancelBtnBsStyle="danger"
+        onConfirm={() => this.hideAlert()}
+      >
+        Invalied Password or Username !
+      </SweetAlert>
+    );
+
+    this.setState({
+      alert: getAlert()
+    });
+  }
+
+  hideAlert() {
+    this.setState({
+      alert: null
+    });
+  }
+
   handleLogin = e => {
     e.preventDefault();
-    this.setState({
-      isLogin: !this.state.isLogin
+    const { username, password } = this.state.loginInfo;
+    const data = JSON.stringify({
+      username,
+      password
     });
+    fetch("/login_post", {
+      credentials: "same-origin",
+      headers: {
+        "content-type": "application/json"
+      },
+      method: "POST",
+      body: data
+    })
+      .then(response => response.json(data))
+      .then(data => {
+        if (data.doneLogin) {
+          this.setState({
+            isLogin: true
+          });
+        } else {
+          this.setState({
+            isLogin: false
+          });
+          this.errorPassOrUsername();
```

Comment:
Are u sure that these 4 lines (61-64) work correctly? Please be assured and get back to me on that.